### PR TITLE
Add gauges to monitor consecutive store refresh failures.

### PR DESCRIPTION
**Pull request by haitaottd**

### CODE REVIEW

**Comment by atarassov-ttd** [comment]
How are you planning to use the new Gauges?

**Comment by haitaottd** [comment]
I plan to observe the metric to pick a threshold for alert.

File: src/main/java/com/uid2/shared/vertx/CloudSyncVerticle.java [hunk 1]
```diff
@@ -64,13 +64,15 @@
     private final Counter counterUploaded;
     private final Counter counterDownloadFailures;
     private final Counter counterUploadFailures;
+    private final Gauge gaugeConsecutiveRefreshFailures;
```
[the remark this answers is not in the document]
Comment (haitaottd) [reply src/main/java/com/uid2/shared/vertx/CloudSyncVerticle.java:67]: This monitors back-to-back failures. Once refresh succeeds, the value is reset to 0. Counter value can only increase, not reset

File: src/main/java/com/uid2/shared/vertx/CloudSyncVerticle.java [hunk 2]
```diff
@@ -64,13 +64,15 @@
...
 
     private final String name;
     private final ICloudStorage cloudStorage;
     private final ICloudStorage localStorage;
     private final ICloudSync cloudSync;
     private final int downloadThreads;
     private final int uploadThreads;
+    private int consecutiveRefreshFailures = 0;
```
[the remark this answers is not in the document]
Comment (haitaottd) [reply src/main/java/com/uid2/shared/vertx/CloudSyncVerticle.java]: yes

**Comment by haitaottd** [comment]
ping

**Pull request merged**